_pytest/logging.py

```diff
@@ -2,15 +2,73 @@
 
 import logging
 from contextlib import closing, contextmanager
+import re
 import six
 
 import pytest
 import py
 
 
+LOGLEVEL_COLOROPTS = {
+    logging.CRITICAL: {'red'},
+    logging.ERROR: {'red', 'bold'},
+    logging.WARNING: {'yellow'},
+    logging.WARN: {'yellow'},
+    logging.INFO: {'green'},
+    logging.DEBUG: {'purple'},
+    logging.NOTSET: set(),
+}
+
 DEFAULT_LOG_FORMAT = '%(filename)-25s %(lineno)4d %(levelname)-8s %(message)s'
 DEFAULT_LOG_DATE_FORMAT = '%H:%M:%S'
 
+# move to __init__ of ColoredLevelFormatter?
+LEVELNAME_FMT_REGEX = re.compile(r'%\(levelname\)([+-]?\d*s)')
+
+
+class ColoredLevelFormatter(logging.Formatter):
+    """
+    Colorize the %(levelname)..s part of the log format passed to __init__.
+    """
+
+    def __init__(self, *args, **kwargs):
+        super(ColoredLevelFormatter, self).__init__(
+            *args, **kwargs)
+        if py.std.sys.version_info[0] >= 3:
```

Comment:
Please use `six.PY3` instead (this will make it easier later to find python2-specific code)